backend/src/services/userService.js

```
@@ -86,8 +88,44 @@ const checkUserVerification = async (userId) => {
+  const getLocationInfoByKey = async (locationKey) => {
+    try {
+      const response = await axios.get(`${config.weather.apiUrlForInfo}/${locationKey}`, {
+        params: {
+          details: true,
+          apikey: config.weather.apiKey,
+        }
+      });
+  
+      return response.data;  
+    } catch (error) {
+      throw new Error('Error fetching location info: ' + error.message);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)